styles.css

```diff
@@ -153,7 +153,16 @@ main .button-section button:active {
 }
```

Comment:
- [ ] [OPTIONAL] Good job so far! In my opinion You need  to pay attention to your `buttons` with your coding partner because `Accessibility` result should at least 90%, you can see error below the picture 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/214296624-33ea6106-a3c0-4346-bd8a-1bba4f6cfb17.png)